### DISCUSSION TITLE
jsh: permission denied: vite

### DISCUSSION THREAD

**Discussion by noam-honig** [stackblitz/tutorialkit#187] [Q&A] [answered]
I was playing around with the configuration and placed the terminal configuration code sample inside a meta.md file  with type: chapter.

I started getting this error - jsh: permission denied: vite

I removed what I've added - and restarted the server - but I still get that error.
I deleted node_modules and all browser cache - and still get this error - I can't seem to find how to get back to normal.

Currently my src folder is identical to the default one, other than the templace which I replaced with something I was working on - and runs.

Is there some hidden cache folder that I can delete?

Here's my repo:
https://github.com/noam-honig/play-with-tutorial-kit

**Comment by AriPerkkio** [comment]
Could you write down reproduction steps for this issue? Starting from `npm create tutorial`, and then showing which `meta.md` files to replace with what content?

The linked Github repository runs just fine.

**Comment by noam-honig** [comment, marked as answer]
Hi - based on your tip - I downloaded the project to a different directory and it worked.
I decided to compare the directories and found the cause.
In the src/template/default there was a node_modules folder (when I tested that it worked)
With that node_modules the problem is reproduced.

to reproduce
`npm create tutorial`
cd src/templates/default
npm i
cd ../../../
npm run dev
you'll see the problem

**Reply by AriPerkkio** [reply]
Thanks, I've created an issue for this: https://github.com/stackblitz/tutorialkit/issues/189

**Reply by noam-honig** [reply]
Perfect - thank you

**Reply by AriPerkkio** [reply]
This is now fixed in `0.1.2` versions. Now `node_modules` of templates are not copied to lesson code. It's safe to have `node_module` present on file system for testing purposes.
